lms/envs/common.py

```diff
@@ -345,9 +345,12 @@
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
 
-    # Show Language selector.
+    # Show language selector in header.
     'SHOW_LANGUAGE_SELECTOR': False,
 
+    # Show language selector in footer.
+    'SHOW_FOOTER_LANGUAGE_SELECTOR': False,
```

Comment:
@andy-armstrong I'm a little hesitant to change the existing setting as I fear doing so would not be backwards compatible and may unintentionally break things for upgrading openedx users. I'm happy to change this, but I want to make sure that we do it in a way that will not cause problems for people. Do you have any suggestions for how this could be done?